packages/generator/tests/acceptance/test_measurement_plugin_client.py

```diff
@@ -0,0 +1,74 @@
+import importlib.util
+import pathlib
+from types import ModuleType
+from typing import Generator, Union
+
+import pytest
+from ni_measurement_plugin_sdk_service.measurement.service import MeasurementService
+
+from ni_measurement_plugin_sdk_generator.client import create_client
+
+
+def test___measurement_plugin_client___measure___returns_output(
+    measurement_plugin_client_module: ModuleType,
+) -> None:
+    test_measurement_client = getattr(measurement_plugin_client_module, "TestMeasurementClient")
+    output = getattr(measurement_plugin_client_module, "Output")
+
+    expected_measure_output = output(
+        float_out=0.05999999865889549,
+        double_array_out=[0.1, 0.2, 0.3],
+        bool_out=False,
+        string_out="sample string",
+        string_array_out=["String1", "String2"],
+        path_out="path/test",
+        path_array_out=["path/test1", "path/ntest2"],
+        io_out="resource",
+        pin_array_out=["pin1", "pin2"],
+        integer_out=10,
+        xy_data_out=None,
+    )
+
+    measurement_plugin_client = test_measurement_client()
+    measure_output = measurement_plugin_client.measure()
+
+    assert measure_output == expected_measure_output
```

Comment:
Arrange the test body into 3 sections separated by blank lines: arrange, act, assert. Do not add comments. These 3 sections correspond to the 3 clauses in the test name (given, when, then).

This applies to all the tests, not just the one I highlighted.
```suggestion
def test___measurement_plugin_client___measure___returns_output(
    measurement_plugin_client_module: ModuleType,
) -> None:
    test_measurement_client = getattr(measurement_plugin_client_module, "TestMeasurementClient")
    output = getattr(measurement_plugin_client_module, "Output")
    expected_measure_output = output(
        float_out=0.05999999865889549,
        double_array_out=[0.1, 0.2, 0.3],
        bool_out=False,
        string_out="sample string",
        string_array_out=["String1", "String2"],
        path_out="path/test",
        path_array_out=["path/test1", "path/ntest2"],
        io_out="resource",
        pin_array_out=["pin1", "pin2"],
        integer_out=10,
        xy_data_out=None,
    )
    measurement_plugin_client = test_measurement_client()
    
    measure_output = measurement_plugin_client.measure()

    assert measure_output == expected_measure_output
```

Reply:
ok